tests/integration/components/bookmark-form/component-test.js

```diff
@@ -0,0 +1,23 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('bookmark-form', 'Integration | Component | bookmark form', {
+  integration: true
+});
+
+test('it sets focus on the title field when empty', function(assert) {
+  this.set('bookmark', {title: null});
+  this.render(hbs`{{bookmark-form bookmark=bookmark}}`);
```

Comment:
I'm not sure why one has to pass the bookmark here, because it should be the same as from the `this.set` above.